web/static/js/components/idea_submission_form.jsx

```diff
@@ -102,7 +103,9 @@ export class IdeaSubmissionForm extends Component {
           </div>
           <div className="eleven wide field">
             <div className="ui fluid action input">
+              <label htmlFor="idea-input-label" className="visually-hidden">Idea input</label>
```

Comment:
hey @ianmcnally, quick check here. can we swap this `id/htmlFor` combo from `'idea-input-label'` to `'idea-body-input'` to better reflect the markup. thanks!

Reply:
you got it. be871a2